x/poe/bootstrap_integration_test.go

```diff
@@ -24,47 +24,46 @@ import (
 
 func TestIntegrationBootstrapPoEContracts(t *testing.T) {
 	// setup contracts and seed some data
-	ctx, example, _, members := setupPoEContracts(t)
+	ctx, example, ocMembers, apMembers := setupPoEContracts(t)
 
 	ocContractAddr, err := example.PoEKeeper.GetPoEContractAddress(ctx, types.PoEContractTypeOversightCommunity)
 	require.NoError(t, err)
-	tcAdapter := contract.NewTrustedCircleContractAdapter(ocContractAddr, example.TWasmKeeper, nil)
-
-	voters, err := tcAdapter.QueryListVoters(ctx)
+	areAllVotingMembers, err := membersAreAllVotingMembers(ctx, ocMembers, ocContractAddr, example.TWasmKeeper)
 	require.NoError(t, err)
+	assert.True(t, areAllVotingMembers)
 
-	//verify the members list are all voting members
-	notVotingMembers := make(map[string]struct{}, len(members))
-	for _, m := range members {
-		notVotingMembers[m] = struct{}{}
-	}
-
-	for _, v := range voters.Members {
-		delete(notVotingMembers, v.Addr)
-	}
-	assert.Empty(t, notVotingMembers)
+	apContractAddr, err := example.PoEKeeper.GetPoEContractAddress(ctx, types.PoEContractTypeArbiterPool)
+	require.NoError(t, err)
+	areAllVotingMembers, err = membersAreAllVotingMembers(ctx, apMembers, apContractAddr, example.TWasmKeeper)
+	require.NoError(t, err)
+	assert.True(t, areAllVotingMembers)
```

Comment:
You can check here that code_ids are pinned and none of the contract instances are privileged to cover acceptance criteria and have some regression tests.